aggregator/src/recursion/circuit.rs

```diff
@@ -25,14 +25,21 @@ use sv_halo2_base::{
     QuantumCell::Existing,
 };
 
-use crate::param::ConfigParams as BatchCircuitConfigParams;
+use crate::param::ConfigParams as RecursionCircuitConfigParams;
 
 use super::*;
 
+/// Convenience type to represent the verifying key.
 type Svk = KzgSuccinctVerifyingKey<G1Affine>;
+
+/// Convenience type to represent the polynomial commitment scheme.
 type Pcs = Kzg<Bn256, Bdfg21>;
+
+/// Convenience type to represent the accumulation scheme for accumulating proofs from multiple
+/// SNARKs.
 type As = KzgAs<Pcs>;
 
+/// Select condition ? LHS : RHS.
```

Comment:
Maybe we should add some more detail: both this and the below entry are helpers to construct corresponding circuits inside of the `ecc_chip` which is maintained in the `loader` argument.

`select_accumulator` select the `lhs` or `rhs` according to the `condition` and return the cells for ec point represent the result of selection.

`accumulate` put all the `accumulators` in argument into circuit and accumulate them into a new pair of ec point and return it.

It may be better also to note that change the size of the vec of `accumulators` argument would lead to a new circuit (vk changed).